requirements.txt

```diff
@@ -6,7 +6,7 @@ pyarrow>=20.0.0
 matplotlib>=3.10.3
 scikit-learn>=1.6.1
 
-# integrations
+# aws/s3 integration
 s3fs==2025.5.1
 boto3==1.37.3
```

Comment:
lets use `zenml integration install aws s3` in the github action maybe to make sure that the future works too?